docs/ordered-assertions.md

```diff
@@ -65,15 +65,13 @@ public void Should_start_work_within_unit_of_work()
 
     // Assert
     var sequentialCallContext = A.SequentialCallContext();
-    A.CallTo(() => unitOfWorkFactory.BeginWork()).MustHaveHappened().InOrder(sequentialCallContext);
-    A.CallTo(() => usefulCollaborator.JustDoIt()).MustHaveHappened().InOrder(sequentialCallContext);
-    A.CallTo(() => unitOfWork.Dispose()).MustHaveHappened().InOrder(sequentialCallContext);
+    A.CallTo(() => unitOfWorkFactory.BeginWork()).MustHaveHappened()
+        .Then(A.CallTo(() => usefulCollaborator.JustDoIt()).MustHaveHappened())
+        .Then(A.CallTo(() => unitOfWork.Dispose()).MustHaveHappened());
 }
 ```
 
-In the assertion phase of the test we need to create a "sequential call context" that records the order of the calls that it's examined while verifying that they occurred.
-
-Then to do the assertions we use the regular `MustHaveHappened` call, but chain on `InOrder(sequentialCallContext)` so the assertion verifies not only that that call happened, but that it occurred in the right order _relative other calls that have been asserted against the same sequential call context_.
+The regular `MustHaveHappened` calls are made, but the results are chained together using `Then`, which verifies not only that that call happened, but that it occurred in the right order relative other calls that have been asserted in the same chain.

```

Comment:
I'll make the same change in IOrderableCallAssertion.cs and AfterMustHaveHappenedConfiguration.cs